intermediate_source/tensorboard_profiler_tutorial.py

```diff
@@ -0,0 +1,242 @@
+"""
+PyTorch Profiler With TensorBoard
+====================================
+This recipe demonstrates how to use PyTorch Profiler
+to detect performance bottlenecks of the model.
+
+.. note::
```

Comment:
I don't think we need this note, since your tutorial already uses new API, we don't need to highlight the older vs newer API here, users would just use the new API (according to this tutorial) and it's fine